src/tui.rs

```diff
@@ -403,6 +409,19 @@ impl ToipeTui {
         //   - won't hang there waiting for user to type space
         lines.push(Text::from(line.join(" ")).with_faint());
 
+        if lines.len() + self.bottom_lines_len + 1 >= terminal_height as usize {
+            return Err(ToipeError::from(format!(
+                "Terminal height is too short! Toipe requires at least {} lines, got {} lines",
+                lines.len() + self.bottom_lines_len + 1,
```

Comment:
Noticed one more thing. This value should be one extra. When `lines.len() + self.bottom_lines_len + 1` == `terminal_height`. The message will look like `Toipe requires at least 10 lines, got 10 lines`

Reply:
done